data/data/aws/master/outputs.tf

```diff
@@ -1,11 +1,3 @@
-output "subnet_ids" {
```

Comment:
#1121 was also in flight removing this (and I'd [asked][1] for it to remove `cluster_id` too).  But with it still dangling, I'm happy to land this removal here instead :).  @staebler, you'll probably need another rebase once this goes in.

[1]: https://github.com/openshift/installer/pull/1121#discussion_r254975180

Reply:
git will notice it does the same thing and not care that we both have the change.